config/config.go

```diff
@@ -10,15 +10,15 @@ const KafkaTLSProtocolFlag = "TLS"
 
 // Config represents service configuration for dp-search-data-finder
 type Config struct {
-	APIRouterURL               string        `envconfig:"API_ROUTER_URL"`
-	BindAddr                   string        `envconfig:"BIND_ADDR"`
-	ContentUpdatedTopicFlag    bool          `envconfig:"CONTENT_UPDATED_TOPIC_FLAG"`
-	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
-	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
-	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
-	KafkaConfig                KafkaConfig
-	ServiceAuthToken           string        `envconfig:"SERVICE_AUTH_TOKEN"   json:"-"`
-	ZebedeeClientTimeout       time.Duration `envconfig:"ZEBEDEE_CLIENT_TIMEOUT"`
+	APIRouterURL                     string        `envconfig:"API_ROUTER_URL"`
+	BindAddr                         string        `envconfig:"BIND_ADDR"`
+	GracefulShutdownTimeout          time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	HealthCheckCriticalTimeout       time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	HealthCheckInterval              time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	KafkaConfig                      KafkaConfig
+	ServiceAuthToken                 string        `envconfig:"SERVICE_AUTH_TOKEN"   json:"-"`
+	ZebedeeClientTimeout             time.Duration `envconfig:"ZEBEDEE_CLIENT_TIMEOUT"`
+	EnablePublishContentUpdatedTopic bool          `envconfig:"ENABLE_PUBLISH_CONTENT_UPDATED_TOPIC"`
```

Comment:
Just a general standard with lists is to keep them in a logical order, if there isn't an obvious order, sort them alphabetically, so I would suggest moving line 21 up to line 14